topo/node/cptx/cptx.go

```diff
@@ -208,6 +208,36 @@ func (n *Node) ConfigPush(ctx context.Context, r io.Reader) error {
 	return nil
 }
 
+func (n *Node) ResetCfg(ctx context.Context) error {
+	log.Infof("%s resetting config", n.Name())
```

Comment:
nit: lets keep this consistent with the other config related log messages

%s - resetting config